CDCgov/prime-reportstream
# Fixed IN Onboarding Failed Test Messages

This PR fixed the following:

Accession numbers for the failed ones - 3558a700-e5a3-494b-b0b8-bf6ae1004166 9ef9098e-c4b2-4916-b1c2-a5f53aa75851

They are for VDF Unity facility with CLIA - 14D2212079.

Error description is - The code for Ordered Test name is missing. Blank identifiers in segments HL7LabReport.HL7PATIENT_RESULT.ORDER_OBSERVATION.ObservationRequest.UniversalServiceIdentifier


Test Steps:
1. ./gradlew clean package
2. ./prime livd-table-dowload -a -e local
3. copy the pdi-covid-19-fake-in.csv to junk
4. ./prime data --input junk/pdi-covid-19-fake-in.csv --input-schema primedatainput/pdi-covid-19 --output-dir junk --route
5. Verify the output file contain OBR-4 as following:
![image](https://user-images.githubusercontent.com/20068283/177179555-065efc15-f170-43b0-98ac-79149e0d22d9.png)
 
Test File: 
[pdi-covid-19-fake-in.csv](https://github.com/CDCgov/prime-reportstream/files/9040450/pdi-covid-19-fake-in.csv)


## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue
- https://app.zenhub.com/workspaces/onboarding--operations-6166edbd409257001e09f1b2/issues/cdcgov/prime-reportstream/5939

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
